migration/migrator/data/course_tables.sql

```diff
@@ -583,6 +583,7 @@ CREATE TABLE "posts" (
 	"endorsed_by" varchar,
 	"type" int NOT NULL,
   "has_attachment" BOOLEAN NOT NULL,
+  "render_markdown" BOOLEAN NOT NULL DEFAULT 'false',
```

Comment:
can you fixing the spacing here (and the line above) to make things to match the surrounding lines?